src/main/groovy/com/cloudbees/groovy/cps/CpsTransformer.groovy

```diff
@@ -126,11 +129,6 @@ class CpsTransformer extends CompilationCustomizer implements GroovyCodeVisitor
         }
     }
 
-    private <T> List<T> copy(List<T> t) {
```

Comment:
Tried package access

```groovy
<T> List<T> copy(List<T> t) {
```

but that was treated as a syntax error. Could have made it `protected` but the method was so trivial it seemed clearer to just inline it.